```
2022/10/18 16:08:34 user: user: Current requires cgo or $USER set in environment
2022/10/18 16:08:34 $USER variable is empty or not set
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x112ab63]
```

--- a/pkgs/applications/file-managers/lf/default.nix
+++ b/pkgs/applications/file-managers/lf/default.nix
@@ -1,4 +1,9 @@
-{ buildGoModule, fetchFromGitHub, lib, installShellFiles }:
+{ lib
+, stdenv
+,buildGoModule
+,fetchFromGitHub
+,installShellFiles
+}:
 
 buildGoModule rec {
   pname = "lf";
@@ -16,6 +21,10 @@ buildGoModule rec {
   nativeBuildInputs = [ installShellFiles ];
 
   ldflags = [ "-s" "-w" "-X main.gVersion=r${version}" ];
+
+  # Force the use of the pure-go implementation of the os/user library.
+  # Relevant issue: https://github.com/gokcehan/lf/issues/191
+  tags = lib.optionals (!stdenv.isDarwin) [ "osusergo" ];
 
   postInstall = ''
     install -D --mode=444 lf.desktop $out/share/applications/lf.desktop

--- a/pkgs/applications/file-managers/lf/default.nix
+++ b/pkgs/applications/file-managers/lf/default.nix
@@ -1,8 +1,8 @@
 { lib
 , stdenv
-,buildGoModule
-,fetchFromGitHub
-,installShellFiles
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
 }:
 
 buildGoModule rec {

